modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsFileDownloadFieldMerge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\media\Entity\Media;
+
+/**
+ * Configure Behavior for paragraphs.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "paragraphs_file_download_field_merge"
+ * )
+ */
+class ParagraphsFileDownloadFieldMerge extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Merging the data into one field.
+    $field_uaqs_download_name = $row->getSourceProperty('field_uaqs_download_name');
+    $field_uaqs_download_description = $row->getSourceProperty('field_uaqs_download_description');
+
+    $value['uaqs_text'] = '<div class="border-thick border-top border-azurite"><div class="border card-body">
+      <h3>' . $field_uaqs_download_name[0]['value'] . '</h3>';
+
+    // Media embeded for field_uaqs_download_file.
+    if (isset($value[0]) && count($value[0])) {
+      foreach ($value[0] as $mid) {
+        $media = Media::load($mid);
+        $value['uaqs_text'] .= '<drupal-media data-align="center" data-entity-type="media" data-entity-uuid="' . $media->get('uuid')->value . '" data-view-mode="default"></drupal-media>';
+      }
+    }
+
+    // Media embeded for field_uaqs_download_preview.
+    if (isset($value[1]) && count($value[1])) {
+      foreach ($value[1] as $mid) {
+        $media = Media::load($mid);
+        $value['uaqs_text'] .= '<drupal-media data-align="center" data-entity-type="media" data-entity-uuid="' . $media->get('uuid')->value . '" data-view-mode="default"></drupal-media>';
+      }
+    }
+    $value['uaqs_text'] .= $field_uaqs_download_description[0]['value'] . '</div></div>';
+    return $value['uaqs_text'];
```

Comment:
Is `uaqs_text` referenced elsewhere in this migration?

Reply:
No @tadean . I just give an array key to hold the entire text for the az_text.

Reply:
@tadean  Please suggest what you would like me to do here?

Reply:
@smitahantzios `uaqs_text` seems a little confusing because it is not present in the migration and does not exist in the `uaqs_file_download` paragraph type or its destination. Perhaps use something like `$value['markup']` instead to make it clear what this placeholder array key is being used for in this process plugin and is not a machine name (Or using a local variable would be okay too).

Reply:
ok sure. Done the change.